action.yaml

```diff
@@ -0,0 +1,29 @@
+name: 'Iter8 Install'
+description: 'Install Iter8 CLI'
+inputs:
+  version:
+    description: "Version of iter8 to install"
+    required: false
+runs:
+  using: 'composite'
+  steps:
+    - name: Install Go
```

Comment:
What is the need for this step?

Reply:
good catch. Initially I used `go install`

Reply:
done.